### ISSUE TITLE
Protocol icons....

### ISSUE THREAD

**Issue by kimme** [EionRobb/pidgin-opensteamworks#155]
Protocol icons are not copied to /usr/share/pixmaps/pidgin/protocols/ this has to be done manually, and the icons are terrible and it lacks the .svg image...

**Comment by xvitaly** [comment]
> Protocol icons are not copied to /usr/share/pixmaps/pidgin/protocols/ this has to be done manually

https://github.com/EionRobb/pidgin-opensteamworks/blob/master/steam-mobile/Makefile#L41

**Comment by kimme** [comment]
I did the install manually as my didn't compile correctly with my Linux Mint 18 build, and updated the steam protocol icons. My icons are here and does include the scalable .svg image

https://www.dropbox.com/s/zi460bid00jjtrt/steam_protocols.zip?dl=0

**Comment by xvitaly** [comment]
> I did the install manually as my didn't compile correctly with my Linux Mint 18 build

I think you installed version from Git tag, which is too outdated (and without icons installation support in makefile). Use version from master instead.

> My icons are here and does include the scalable .svg image

What license of these icons?

**Comment by kimme** [comment]
The base image is taken from http://www.flaticon.com/ and is available here 
http://image.flaticon.com/icons/svg/23/23822.svg

The rest of the images are png exported with Gimp to the correct sizes, and the base image is used as the scalable image.

**Comment by EionRobb** [comment]
There's kind of several issues all wrapped up into one issue report here: 

As mentioned the Makefile installs the icons as part of the "make install" command. 

You say the plugin "didn't compile correctly with my Linux Mint 18 build". Can you clarify what you mean? 

"The icons look ugly" is fairly subjective. The included icons follow the Tango icon spec that all other icons in Pidgin follow 

There is no svg because Pidgin does not use them and I'm not aware of any other libpurple UI that does use them. Which UI are you using libpurple with? 

And finally, the icons you've linked to would be unsuitable to use with the plugin since they are not licensed in a GPL-compatible licence (eg CC) nor do they follow the Tango icon spec

**Comment by kimme** [comment]
My build doesn't compiled because I hadn't "sudo apt install" the correct development files, and I manually installed the .so file to my Pidgin enviroment.

The protocol icons in my opinion are ugly displayed in Pidgin and I used the base image in the webpage I linked to my new protocol images and manually copied them to /usr/share/pixmaps/pidgin/protocols/

**Comment by EionRobb** [comment]
Fair enough, the readme says to just copy the .so without saying which one and without giving instructions about how to install the icons. It's probably worth changing.

**Comment by wandth** [comment]
@xvitaly: Are there any newer builds in master than the 1.61?

**Comment by xvitaly** [comment]
> Are there any newer builds in master than the 1.61?

All Fedora builds can be found at [Koji build system](https://koji.fedoraproject.org/koji/packageinfo?packageID=22538).

**Comment by wandth** [comment]
@xvitaly: Thx! Unfortunately one of the builds work (when I copy the purple-libsteam.so file in my "~/.purple/plugins" folder. As long as I use the x68_64 version, the Fedora builds should also work on Ubuntu, or not?

**Comment by xvitaly** [comment]
> the Fedora builds should also work on Ubuntu, or not

No. You should build from sources on Ubuntu.

**Comment by wandth** [comment]
I was searching for ready builds but thx for the info. Got my own build working.
